CONFIG.H.IN

```diff
@@ -2,5 +2,6 @@
 #define _TK_CONFIG_H_
 
 #cmakedefine TK_DATADIR "@TK_DATADIR@"
+#cmakedefine TK_MACOSX_BUNDLE 1
```

Comment:
Nope, we don't care about the value, just whether or not it's defined.

This will end up with
```c
#define TK_MACOSX_BUNDLE 1
```
or
```c
/* #undef TK_MACOSX_BUNDLE */
```